src/sources/kubernetes_logs/parser/docker.rs

```diff
@@ -42,7 +43,10 @@ fn parse_json(log: &mut LogEvent) -> Option<()> {
             }
             Some(())
         }
-        Ok(_) | Err(_) => None,
+        Ok(_) | Err(_) => {
+            emit!(KubernetesLogsMessageParseFailed { message: &to_parse });
```

Comment:
Looks good, but let's put the `Docker` somewhere in the name. This should be a docker-specific parser error - pretty much mirroring the similar emit at JSON.

How about `KubernetesLogsDockerFormatParseFailed`?

Similarly, the `counter!` can emit `k8s_docker_format_parse_failures`.

Reply:
Sounds good.